backend/src/main/java/com/wire/bots/roman/MessageHandler.java

```diff
@@ -430,6 +435,24 @@ private boolean send(OutgoingMessage message) {
         }
     }
 
+    private boolean shouldIgnore(Provider provider, OutgoingMessage message) {
+        if (message.type.equalsIgnoreCase("conversation.bot_request"))
+            return false;
+
+        if (provider.commandPrefix == null)
+            return false;
+
+        if (provider.commandPrefix.equalsIgnoreCase("***"))
+            return true;
+
+        if (!message.type.equalsIgnoreCase("text"))
```

Comment:
so you want to ignore all assets for example? is that correct?

Reply:
if this is a command prefix bot then yes.. only commands are sent to the bot

Reply:
ok, fair enough